readthedocs_build/config/config.py

```diff
@@ -433,14 +433,10 @@ def load(path, env_config):
     The config will be validated.
     """
 
-    filename = find_one(path, CONFIG_FILENAMES)
+    filename = find_one(path, CONFIG_FILENAME_REGEX)
 
     if not filename:
-        files = '{}'.format(', '.join(map(repr, CONFIG_FILENAMES[:-1])))
-        if files:
-            files += ' or '
-        files += '{!r}'.format(CONFIG_FILENAMES[-1])
-        raise ConfigError('No files {} found'.format(files),
+        raise ConfigError('No files with regex \'{}\' found'.format(CONFIG_FILENAME_REGEX),
```

Comment:
We don't need to mention regex here as this is a user facing error. The error message should probably just be "No configuration file found."

Reply:
Done